.changeset/gentle-feet-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix(types): use arrow functions instead of bound functions
```

Comment:
Applied!

There are also some very minor JavaScript changes in `packages/package`, but since I don't think it would be noticeable to any users of `packages/package`, I've skipped creating a changeset for it.